changelog.rst

```diff
@@ -3,6 +3,23 @@
 Release History
 ===============
 
+0.6.8 (dev)
+------------------
+|new| run configuration parameter to enable relaxation of the `demand_share_per_timestep_decision` constraint.
+
+|new| `storage_cap_min/equals/max` group constraints added.
+
+|changed| Updated to Pyomo 6.2, pandas 1.3, xarray 0.20, numpy 1.20.
+
+|changed| parameters defaulting to False now default to None, to avoid confusion with zero.
+
+|changed| `INFO` logging level includes logs for dataset cleaning steps before saving to NetCDF and for instantiation of timeseries clustering/resampling (if taking place).
+
+|fixed| `demand_share_per_timestep_decision` constraint set includes.
```

Comment:
"included" ?

Reply:
Looks like I forgot to finish the sentence...